lib/query/querycompiler.js

```diff
@@ -892,6 +892,34 @@ class QueryCompiler {
         this.bindingsHolder
       )})`;
     } else {
+      function chunk(arr, start, amount) {
+        var result = [],
+          i,
+          start = start || 0,
+          amount = amount || 500,
+          len = arr.length;
+
+        do {
+          //console.log('appending ', start, '-', start + amount, 'of ', len, '.');
+          result.push(arr.slice(start, start + amount));
+          start += amount;
+        } while (start < len);
+
+        return result;
+    }
+
+    if (Array.isArray(statement.column))
+        return this.multiWhereIn(statement);
+
+    if(statement.value.length > 1000) {
```

Comment:
1000 is a magic number, that need to be explain and documented. The reason is here: 
https://docs.oracle.com/en/database/oracle/oracle-database/12.2/sqlrf/Expression-Lists.html#GUID-5CC8FC75-813B-44AA-8737-D940FA887D1E

"A comma-delimited list of expressions can contain no more than 1000 expressions. A comma-delimited list of sets of expressions can contain any number of sets, but each set can contain no more than 1000 expressions."